snowflake/tests/test_unit.py

```diff
@@ -26,6 +26,24 @@ def test_config():
         SnowflakeCheck(CHECK_NAME, {}, [account_config])
 
 
+def test_default_authentication(instance):
+    # Test default auth
+    check = SnowflakeCheck(CHECK_NAME, {}, [instance])
+    assert check.config.authenticator == 'snowflake'
```

Comment:
(nit) I would favour testing that `sf.connect()` is receiving the correct `authenticator` param using a mock.assert_called_with.

Reply:
Updated!